README.md

```diff
@@ -47,7 +66,8 @@ optional arguments:
 
 Data for the Vitessce visualization almost always comes via raw data that is processed by [ingest-pipeline](https://github.com/hubmapconsortium/ingest-pipeline) airflow dags.
 Harvard often contributes our own custom pipelines to these dags that can be found in [portal-containers](https://github.com/hubmapconsortium/portal-containers).
-The outputs of these pipelines are then converted into view configurations for Vitessce by the [portal backend](https://github.com/hubmapconsortium/portal-ui/blob/0b43a468fff0256a466a3bf928a83893321ea1d9/context/app/api/client.py#L165),
+The outputs of these pipelines are then converted into view configurations for Vitessce by the [portal backend](https://github.com/hubmapconsortium/portal-ui/blob/0b43a468fff0256a466a3bf928a83893321ea1d9/context/app/api/client.py#L165), The `vis-preview.py` mimics the invocation of `get_view_config_builder` for development and testing purposes independently, i.e., without using the [portal backend](https://github.com/hubmapconsortium/portal-ui/blob/0b43a468fff0256a466a3bf928a83893321ea1d9/context/app/api/client.py#L165).
```

Comment:
These links are a bit outdated since the `client.py` file is now in `portal-visualization`: https://github.com/hubmapconsortium/portal-visualization/blob/main/src/portal_visualization/client.py

Both of these links also appear to point to the same URL - I think pointing to where the portal UI hooks into `client.py` for view conf generation might be more illustrative: https://github.com/hubmapconsortium/portal-ui/blob/main/context/app/routes_browse.py#L126